scripts/check-npm.sh

```diff
@@ -1,6 +0,0 @@
-#!/usr/bin/env bash
-
-if [[ `npm -v` != 3* ]]
-  then
-    npm install --global npm
-fi
```

Comment:
Here you go https://github.com/commercetools/nodejs/commit/df446fc597181b37d3b254eeb1298b773eb71d7f